src/Defra.PTS.Web.UI/Views/Shared/_FooterPartial.cshtml

```diff
@@ -30,21 +30,20 @@
                     @if(Thread.CurrentThread.CurrentCulture.EnglishName != "Welsh")
                     {
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" href="" id="localeCy">
+                            <a class="govuk-footer__link" href="#" id="localeCy" onclick="changeLanguage('cy')">
                                 Cymraeg
                             </a>
                         </li>
                     }
                     else
                     {
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" href="" id="localeEn">
+                            <a class="govuk-footer__link" href="#" id="localeEn" onclick="changeLanguage('en-GB')">
```

Comment:
Same here

Reply:
When clicked, it triggers the changeLanguage function with the argument 'en-GB', which stands for English language.